nemo/collections/nlp/modules/common/megatron/utils.py

```diff
@@ -127,6 +127,13 @@ def init_(tensor):
     return init_
 
 
+def init_method_kaiming_uniform():
+    def init_(tensor):
+        return torch.nn.init.kaiming_uniform_(tensor, a=math.sqrt(5))
```

Comment:
Can the value of `a` be a parameter instead of hard coding to `math.sqrt(5)`?

Reply:
fixed